packages/patch/patch.1.0.0/opam

```diff
@@ -12,6 +12,7 @@ depends: [
   "dune"
   "alcotest" {with-test}
   "crowbar" {with-test}
+  "base-bytes" {ocaml:version >= "5.0"}
```

Comment:
Package variables can't be used here (technically for `:version`, opam _could_ rewrite it to a package formula, but it doesn't). I can't edit L11, but this wants to be:
```suggestion
  ("ocaml" {>= "4.04.2" & < "5.0"} | "ocaml" {>= "5.0"} & "base-bytes")
```
without L11 above.